frontend/src/components/institute/NewSchoolPage.vue

```diff
@@ -386,15 +412,16 @@ export default {
       requiredAuthoritySchoolCategories: ['INDEPEND', 'INDP_FNS', 'OFFSHORE'],
       noGradeSchoolCategory: ['POST_SEC', 'EAR_LEARN'],
       districtSchoolCategoryConstraints: [
-        { districtRegionCode: 'OFFSHORE', schoolCategory: 'OFFSHORE' },
-        { districtRegionCode: 'PSI', schoolCategory: 'POST_SEC' },
-        { districtRegionCode: 'YUKON', schoolCategory: 'YUKON' }
+        {districtRegionCode: 'OFFSHORE', schoolCategory: 'OFFSHORE'},
+        {districtRegionCode: 'PSI', schoolCategory: 'POST_SEC'},
+        {districtRegionCode: 'YUKON', schoolCategory: 'YUKON'}
       ],
       schoolCategoryDisabled: false,
       newSchool: {
         districtID: null,
         independentAuthorityId: null,
         displayName: null,
+        legacySafeSchoolName: null,
```

Comment:
This variable is not mapped in node. Use **displayNameNoSpecialChars** instead to match the school object in the api